relay-server/src/actors/store.rs

```diff
@@ -649,6 +649,17 @@ impl Handler<StoreEnvelope> for StoreForwarder {
                 attachments,
             });
 
+            if self
+                .config
+                .processing_internal_projects()
+                .contains(&scoping.project_id.value())
+            {
+                metric!(
+                    counter(RelayCounters::InternalCapturedEventStoreActor) += 1,
+                    event_item_type = &event_item.ty().to_string(),
```

Comment:
Since there's a low number of projects, how about adding `project = scoping.project_id.value()` as a tag?

Reply:
it's just going to be `[1]` but I added it